src/components/collection_vote_box/CollectionSubmitVoteBox.js

```diff
@@ -48,7 +71,7 @@ export default function CollectionSubmitVoteBox(props) {
   const {question, votes, isFinal, isApprovalDisabled, isLoading, adminPage} = props;
 
   useEffect(() => {
-    setIsVotingDisabled(props.isDisabled || (isFinal && submitted) || isLoading);
+    setIsVotingDisabled( props.isDisabled || (isFinal && submitted) || adminPage);
   }, [props.isDisabled, isFinal, submitted, isLoading]);
```

Comment:
Nit: Add `adminPage` to the dependency array. Even if the value doesn't change after component load, adminPage is a prop and the `useEffect` depends on its value.